### PR TITLE
bt-24: AWS deployment

### LINKED ISSUE #24: AWS Deployment

### PR DESCRIPTION
**Pull request by MaksymGavatuk**

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: Continuous Delivery
+
+env:
+  AWS_REGION: us-east-1
+  EBS_APP_NAME: bsa-talents
+  ENVIRONMENT: ${{ github.ref_name }}
```
Comment (katerynakostikova) [inline .github/workflows/cd.yml]:
```suggestion
env:
    AWS_REGION: us-east-1
    EBS_APP_NAME: bsa-talents
    ENVIRONMENT: ${{ github.ref_name }}
```
and so on

Comment (MaksymGavatuk) [reply .github/workflows/cd.yml]: Oh yes 😅 Thanks

File: prepare-build.sh
Comment (nikita-remeslov) [inline prepare-build.sh:1]: eol

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
we should update GitHub Actions, they require npm 9

**Review by nikita-remeslov — APPROVED** [review]

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**